projects/rulesets-base/src/extended-rules/spectral-rule.ts

```diff
@@ -54,6 +64,15 @@ function toOpticResult(
   };
 }
 
+// This will treat info + hint as a info in Optic
+function severityToOpticSeverity(spectralSev: DiagnosticSeverity): Severity {
+  return spectralSev === DiagnosticSeverity.Error
```

Comment:
looks good. If I'm reading this right it looks like we only support `error` `warn` and `info`? seems reasonable.

Reply:
Yes thats correct